R/route.R

```diff
@@ -99,28 +99,37 @@ route_dodgr <-
   }
 
   # Try to get route network if net not provided
-  pts <- rbind(from_coords, to_coords)
   if(is.null(net)) {
-    net <- dodgr::dodgr_streetnet(pts = pts, expand = 0.2)
+      pts <- rbind(from_coords, to_coords)
+      net <- dodgr::dodgr_streetnet(pts = pts, expand = 0.2)
   }
 
-  suppressWarnings(
-    suppressMessages(
-      ways_dg <- dodgr::weight_streetnet(net)
-      )
+  suppressMessages (
+    ways_dg <- dodgr::weight_streetnet(net)
   )
 
   verts <- dodgr::dodgr_vertices(ways_dg) # the vertices or points for routing
-  knf <- nabor::knn(cbind(verts$x, verts$y), matrix(from_coords, ncol = 2), k = 1)
-  knt <- nabor::knn(cbind(verts$x, verts$y), matrix(to_coords, ncol = 2), k = 1)
-  dp <- dodgr::dodgr_paths(ways_dg, from = verts$id[knf$nn.idx], to = verts$id[knt$nn.idx])
-  path <- verts[match(dp[[1]][[1]], verts$id), ]
-  path_linestring <- sf::st_linestring(cbind(path$x, path$y))
-  path_sf <- sf::st_sf(sf::st_sfc(path_linestring), crs = 4326)
-  # plot(net$geometry) # in there for the fun of it (to be removed)
-  # plot(path_linestring, col = "red", lwd = 5, add = TRUE)
-  path_sf
-
-  # clever routing code
-
+  #suppressMessages ({
```

Comment:
Note that I think these messages are useful:
``` r
r <- route_dodgr(from, to, net = osm_net_example)
#> xy has no named columns; assuming order is x then y
#> xy has no named columns; assuming order is x then y
```
Admittedly somewhat confusing because of the `xy`, but that indicates that neither `from` nor `to` have named columns, and so are presumed `(x, y)`.